frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -94,6 +113,33 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
       .finally(() => setDownloading(false));
   };
 
+  const handleFullScreen = () => {
+    setIsFullScreen(!!document.fullscreenElement);
+  };
+
+  useEffect(() => {
+    document.addEventListener('fullscreenchange', handleFullScreen);
+    return () => {
+      document.removeEventListener('fullscreenchange', handleFullScreen);
+    };
+  }, [isFullScreen]);
```

Comment:
This will recognize, if the user uses ESC key/keyboard keys to exit the full screen mode.